src/ducks/apps/components/Application.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
```

Comment:
This file need to be renamed as well. The filename must exactly with the component class name.